data_scripts/get_macros.py

```diff
@@ -64,6 +64,8 @@
             macros.stonefly = row[10]
             macros.water_penny = row[11]
             macros.dobsonfly = row[12]
+            for i in range(7, 12+1):
+                macros.sensitive_total += int(row[i])
```

Comment:
Are you just adding the values together to get the macro totals? If so, the ``sensitive_total`` and ``somewhat_sensitive_total`` should be weighted.